packages/release-brancher/src/release-brancher.ts

```diff
@@ -245,7 +245,14 @@ export class Runner {
       }
     }
 
-    const message = `build: configure branch ${this.branchName} as a release branch`;
+    // java-lts was for June 2021 release. Use java-lts-no-sp for the December 2021 release onwards.
+    const withSpVersion = this.releaseType === 'java-lts';
+    // This relies on "feat: " keyword to bump the next minor version from default branch.
+    // Alternatively we could use "Release-As: XXX" keyword but that would require fetching the
+    // version of the latest release. "feat: " is more simple to do the same task.
```

Comment:
@chingor13 Thank you for several ideas the other day. Here I added a note for the idea I didn't use.